go/private/rules/cgo.bzl

```diff
@@ -212,56 +205,3 @@ def _include_unique(opts, flag, include, seen):
         return
     seen[include] = True
     opts.extend([flag, include])
-
```

Comment:
A handful of people are probably using this, but it's undocumented, so it's probably okay to remove it.

I think users will just need to depend directly on the `go_binary` target instead of the `.cc` target, right?

Reply:
Yeah. Actually I'm going to add an alias that's created regardless (so that it doesn't have to use linkmode as a macro).

Reply:
I have added a `.cc` alias to all go_binary with a `manual` tag and a deprecation message just in case. I can remove it but I felt it harmless.

Reply:
That sounds like a good approach. Let's also mention this in https://github.com/bazelbuild/rules_go/wiki/Deprecation-schedule and say that it will be removed in v0.27.0.